packages/mobile/src/paymentRequest/PaymentRequestConfirmation.tsx

```diff
@@ -141,7 +132,7 @@ class PaymentRequestConfirmation extends React.Component<Props> {
             recipient={recipient}
             address={requesteeAddress || ''}
             e164PhoneNumber={recipient.e164PhoneNumber}
-            comment={reason}
+            // comment={reason} NOTE: Add this back in when redesigning this screen
```

Comment:
Ditto

Reply:
Payment screens aren't included in this PR